assets/js/pages/ClusterDetails/AscsErsClusterDetails.jsx

```diff
@@ -195,6 +195,10 @@ function AscsErsClusterDetails({
                 title: 'CIB last written',
                 content: cibLastWritten || '-',
               },
+              {
```

Comment:
@jagabomb Is this the correct position? I guess you can get the view in storybook

Reply:
@CDimonaco By the way, seeing `Cluster Maintenance` with that capital `M` looks out of place, as the rest of the titles don't have capital letters except the first word. I would name it `Cluster maintenance`, and the same for the hana cluster details

Reply:
And yet another thing. I think we should place the maintenance mode in other place.
The `Cluster type` should go in the top/right corner, as it was before, and put the `Maintenance mode` in the left buttom corner, to keep consistency with the hana details view

Reply:
done!